gunicorn_config.py

```diff
@@ -24,12 +20,8 @@
 if not running_dev():
     # Saves memory in the worker process, but breaks --reload
     preload_app = True
-    # avoid restarting gunicorn and leave it to pod restarts to handle memory leaks
-    max_requests = 0
```

Comment:
this is set here for a reason ... we do not want gunicorn to restart ... as it results in Bad Gateway errors for in flight http responses being served ... if there are memory leaks (which we should not have) let pod deal with it